cmd/cmd.go

```diff
@@ -16,26 +16,36 @@ import (
 )
 
 const (
+	// Use is the name of the CLI application.
 	Use = "dynatrace-bootstrapper"
 
-	SourceFolderFlag   = "source"
-	TargetFolderFlag   = "target"
-	DebugFlag          = "debug"
+	// SourceFolderFlag is the flag for specifying the source folder.
+	SourceFolderFlag = "source"
+	// TargetFolderFlag is the flag for specifying the target folder.
+	TargetFolderFlag = "target"
+	// DebugFlag is the flag for enabling debug mode.
+	DebugFlag = "debug"
+	// SuppressErrorsFlag is the flag for suppressing errors.
```

Comment:
These comments are just noise

the linter for enforcing this should be removed.

Comments make sense for complicated/not-obvious things, IMO they have to answer a "WHY" question, rather then describing things just with more words. (these all sound like, "A Mug is a mug")

Best example: `DebugFlag is the flag for enabling debug mode.`
- what is "debug mode"?
- btw, this just enables the debug logs, nothing, more